api/src/main/java/jakarta/data/metamodel/restrict/BasicRestrictionRecord.java

```diff
@@ -39,4 +39,24 @@ public BasicRestriction<T> negate() {
                 comparison.negate(),
                 value);
     }
+
+    /**
+     * Textual representation of a basic restriction.
+     * For example,
+     * <pre>price LESS_THAN 50.0</pre>
+     *
+     * @return textual representation of a basic restriction.
+     */
+    @Override
+    public String toString() {
+        StringBuilder s = new StringBuilder();
```

Comment:
If possible, avoid variables like `s`, please, use something like: 

`textExpression`

or any variation like this.

Reply:
I renamed it in the latest commit.